src/subscript/sector2fluxnum/flux_obj.py

```diff
@@ -0,0 +1,275 @@
+import sys
+import os
+
+from ecl.eclfile import EclKW
+from ecl.grid import EclRegion
+
+try:
+    from ecl import EclDataType
+except ImportError:
+    from ecl import EclTypeEnum
+
+import subscript.sector2fluxnum.flux_util as futil
```

Comment:
Should probably not be shortened to less than `flux_util` for readability.

Reply:
ok